File: Two Sum/solution2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}
        length = len(nums)
        for i in range(length):
            k = target - nums[i]
            if k in hashmap:
                return [i, hashmap[k]]
            hashmap[nums[i]] = i

File: Two Sum/solution2.go
func twoSum(nums []int, target int) []int {
    var length int = len(nums) - 1
    hashmap := make(map[int]int)
    for i:= 0; i <= length; i++ {
        var k int = target - nums[i]
        if value, ok := hashmap[k]; ok {
            return []int{i, value}
        }
        hashmap[nums[i]] = i
    }
    return []int{}
}
